packages/snap-store-mobx/src/Search/Stores/SearchResultStore.ts

```diff
@@ -741,10 +735,13 @@ export class Variant {
 	constructor(variantData: { data: { variant: VariantData } }) {
 		const { data } = variantData || {};
 		const { variant } = data || {};
-		this.attributes = variant.attributes;
+		this.attributes = variant.attributes || {};
 		this.mappings = variant.mappings;
 		this.options = variant.options;
-		this.available = (this.attributes.available as boolean) || false;
+		// construct badges from data (need meta)
+		this.badges = variant.badges || [];
+
+		this.available = (this.attributes.available as boolean) ?? (this.mappings.core?.available as boolean) ?? true;
```

Comment:
Seems we have changed the fallback to `true` when it was `false` before?  Is that right? 🤔